### ISSUE TITLE
File path wrongly referenced on Windows in built project

### ISSUE THREAD

**Issue by jedchang** [VladimirMikulic/parcel-plugin-custom-dist-structure#17]
Inserting pictures in HTML is normal under dev and build mode. But if the image is changed to SCSS, the dev is normal, but the build mode cannot be displayed?
Looks like a picture path problem?

 - OS:  Windows 10
 - Parcel version: v1.12.4
 - Plugin version: v1.1.9

package.json
```json
{
  "name": "demo-parcel",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "keywords": [],
  "author": "",
  "license": "ISC",
  "scripts": {
    "clean": "rm dist/index.js",
    "start": "rm -rf ./dist && parcel ./src/index.html",
    "build": "rm -rf ./dist && parcel build ./src/index.html --public-url ./"
  },
  "customDistStructure": {
    "config": {
      "js": [
        ".js",
        ".js.map"
      ],
      "css": [
        ".css",
        ".css.map"
      ],
      "images": [
        ".jpg",
        ".png"
      ]
    },
    "options": {
      "development": true
    }
  },
  "dependencies": {},
  "devDependencies": {
    "@babel/core": "^7.9.0",
    "@babel/preset-env": "^7.9.5",
    "autoprefixer": "^9.7.6",
    "parcel-bundler": "^1.12.4",
    "parcel-plugin-custom-dist-structure": "^1.1.9",
    "postcss-modules": "^2.0.0",
    "sass": "^1.26.3"
  }
}

```

index.html
```html
<!DOCTYPE html>
<html>
  <head>
    <title>Empty project</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="./scss/main.scss" />
  </head>
  <body>
    <div id="app">
      <div id="banner">
        <img src="./images/banner-bg.jpg" alt="" />
      </div>
    </div>
    <script src="./js/main.js"></script>
  </body>
</html>
```

main.scss
```scss
#banner {
  width: 100%;
  height: 600px;
  background-image: url('../images/banner-bg.jpg');
}
```

preview

![q-all](https://user-images.githubusercontent.com/8979857/80443083-f89a5e80-8940-11ea-8f72-23d8552825dd.jpg)

**Comment by VladimirMikulic** [comment]
Hi @jedchang. Thank you for reporting this issue.
The fix will be published soon.

**Comment by VladimirMikulic** [comment]
Hi @jedchang. Sorry for a late reply, I was busy this week.
I think I've fixed it, but I'll need your help to verify this :)

I would like you to test it, the steps are:
1. Clone the project: `git clone https://github.com/VladimirMikulic/parcel-plugin-custom-dist-structure.git`
2. Check out the test branch: `git checkout fix/incorrect-windows-file-paths`
3. Make it available globally on your system: `npm link`
4. After running npm link, you'll get the absolute path to the package -> copy it
5. Enter your project and run npm i -D <absolute_path> to install the plugin (before this, make sure to delete the version from NPM)
6. Report the results here

Thank you.

**Comment by VladimirMikulic** [comment]
By the way, instead of doing `rm -rf`, you can use `parcel-plugin-nuke-dist` (not my plugin, just a suggestion :)

**Comment by jedchang** [comment]
> Hi @jedchang. Sorry for a late reply, I was busy this week.
> I think I've fixed it, but I'll need your help to verify this :)
> 
> I would like you to test it, the steps are:
> 
> 1. Clone the project: `git clone https://github.com/VladimirMikulic/parcel-plugin-custom-dist-structure.git`
> 2. Check out the test branch: `git checkout fix/incorrect-windows-file-paths`
> 3. Make it available globally on your system: `npm link`
> 4. After running npm link, you'll get the absolute path to the package -> copy it
> 5. Enter your project and run npm i -D <absolute_path> to install the plugin (before this, make sure to delete the version from NPM)
> 6. Report the results here
> 
> Thank you.

Hi @VladimirMikulic, 
Follow the steps you provided to test, the path of the picture becomes `url (..\images\banner-bg.de802b11.jpg)`
But the same picture is not displayed? It seems to be changed to `url (../images/banner-bg.de802b11.jpg)` to display?

![04_x_04_11-05-48](https://user-images.githubusercontent.com/8979857/80933929-58d54880-8df8-11ea-942e-9624e974143c.jpg)

**Comment by VladimirMikulic** [comment]
Hi @jedchang. Thank you for testing it.
I've pushed a new change, could you please try it out following the same steps from above?

Thank you.

**Comment by jedchang** [comment]
HI @VladimirMikulic 
Awesome! after testing, the path worked perfectly. 
Thank you~

**Comment by VladimirMikulic** [comment]
Awesome! Thank you very much for helping me.
This patch will be published together with the patch for #19.

**Comment by VladimirMikulic** [comment]
I have good news. The v1.1.10 has been published and includes this patch.
Enjoy :rocket:
